pkg/test/vpc_client/bastion.go

```diff
@@ -155,3 +178,121 @@ func (vpc *VPC) DestroyBastionProxy(instance types.Instance) error {
 	}
 	return nil
 }
+
+func generateBcryptPassword(plainPassword string) (string, error) {
+	hashedPassword, err := bcrypt.GenerateFromPassword([]byte(plainPassword), bcrypt.DefaultCost)
+	if err != nil {
+		log.LogError("Generate hashed password failed")
+		return "", nil
+	}
+	return string(hashedPassword), nil
+}
+
+func writePasswordToFile(username, hashedPassword, filePath string) error {
```

Comment:
Is there a common func defined for write file?

Reply:
Use Ying's run ssh command to set password for bastion proxy, local file store password is not needed. This part has been removed.